tests/actions/patient.test.js

```diff
@@ -33,6 +33,19 @@ describe('Patient actions', () => {
     done();
   });
 
+  it('fetch patient records failure case', async (done) => {
+    const { defaultPatient } = mockData;
+    let request = moxios.requests.mostRecent();
+    moxios.stubRequest(`${apiBaseUrl}/patient/${uuid}?v=custom:(patientId,uuid,patientIdentifier:(uuid,identifier),person:(gender,age,birthdate,birthdateEstimated,personName,preferredAddress),attributes:(value,attributeType:(name)))`, {
+      status: 400,
+      response: defaultPatient
+    });
+  
+    const store = mockStore({});
+    await store.dispatch(fetchPatientRecord(uuid));
+    done();
+  });
```

Comment:
there is no assertion for this test-case   😉.

Reply:
Yes and try to use proper test descriptions that make it easy to understand what the test is doing.

Reply:
Done